docs/deployment.md

```diff
@@ -74,73 +71,38 @@ Next.js can be deployed to any hosting provider that supports Node.js. Make sure
 }
 ```
 
-`next build` builds the production application in the `.next` folder. After building, `next start` starts a Node.js server that supports [hybrid pages](/docs/basic-features/pages.md), serving both statically generated and server-side rendered pages.
+Next, run `next build` to build your application. Finally, run `next start` to start the Node.js server. This server supports every feature of Next.js.
```

Comment:
```suggestion
Then, run `next build` to build your application. Finally, run `next start` to start the Node.js server. This server supports every feature of Next.js.
```

Just trying to avoid people thinking it's a pun

Reply:
```suggestion
Next, run `next build` to build your application. Finally, run `next start` to start the Node.js server. This server supports all Next.js features.
```

Feel free to discard this one, just a small comment.